### PR TITLE
Migrate :norway: databases on deploy 

### PR DESCRIPTION
**Pull request by lorentzlasson**
#530 

---

Tested on devices

- [ ] Desktop 💻
- [ ] Mobile 📱

Tests

- [ ] All tests are running ✔️
- [ ] Test are updated 🧪
- [ ] Code Review 👩‍💻
- [ ] QA 👌

⏲️ Time spent on CR:

⏲️ Time spent on QA:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -502,14 +502,30 @@ npx prisma migrate dev --name <migration-name>
 
 This will create a new migration file in [/prisma/migrations](prisma/migrations).
 
-Currently these migrations are not automatically run in our CI/CD pipeline, so it has to be done manually:
-
-1. Run the [cloud sql auth proxy locally](#google-cloud--cloud-sql-auth-proxy-setup)
-2. Update your environment variables to point to the production database.
-3. Run `npx prisma migrate deploy` to apply the migration to the production database.
```
Comment (lorentzlasson) [inline README.md:509]: No need to keep this moving forward right?

Comment (adamalfredsson) [reply README.md:509]: Yay 🎉

File: README.md [hunk 2]
```diff
@@ -502,14 +502,30 @@ npx prisma migrate dev --name <migration-name>
...
+gcloud run jobs create <SERVICE_NAME>-db-migrate \
+  --image gcr.io/<PROJECT_ID>/<IMAGE_NAME>:latest \
+  --set-secrets DB_URL=<SERVICE_NAME>-db-url:latest \
+  --set-cloudsql-instances <PROJECT_ID>:europe-north1:<CLOUD_SQL_NAME> \
+  --region europe-north1 \
+  --max-retries 0 \
+  --command "./node_modules/.bin/prisma" \
+  --args "migrate,deploy"
+```
+
```
Comment (adamalfredsson) [inline README.md:528]: Nice!

**Review by adamalfredsson — APPROVED** [review]
We should make sure that the migrations are marked as applied on the production database before proceeding with this!

From [prisma docs](https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/baseline-your-database-typescript-postgresql#review-the-migration):

> Next, mark the migration as applied using prisma migrate resolve with the --applied argument.
> 
> `npx prisma migrate resolve --applied 0_init`
> The command will mark 0_init as applied by adding it to the _prisma_migrations table.

**Pull request merged**